### PR TITLE
card component

### PR DESCRIPTION
**Pull request by rohanmathur91**
Please review the card component.

### CODE REVIEW

File: documentation/style.css [hunk 1]
```diff
@@ -156,7 +157,7 @@ p {
 	color: var(--dark-gray);
 }
 
-@media screen and (max-width: 700px) {
+@media screen and (max-width: 800px) {
 	.logo {
 		display: none;
```
Comment (smilegupta) [inline documentation/style.css]: which screen are your targeting here by mentioning 800px?

File: documentation/style.css [hunk 2]
```diff
@@ -202,7 +199,7 @@ p {
 	.sidebar {
 		position: fixed;
 		top: 0;
-		z-index: 5;
+		z-index: 15;
 		width: 0rem;
 		height: 100%;
```
Comment (smilegupta) [inline documentation/style.css]: why such a big z-index?

File: index.html [hunk 3]
```diff
@@ -15,7 +15,7 @@
 </head>
 
 <body>
-    <nav class="navigation spacing flex-row items-center wrap py-2 px-6 fixed left-0 top-0 z-2 w-100">
+    <nav class="navigation spacing flex-row items-center wrap py-2 px-6 fixed left-0 top-0 z-10 w-100">
```
Comment (smilegupta) [inline index.html:18]: what is z property?

**Review by smilegupta — COMMENTED** [review]
Added a few comments on the PR

File: documentation/style.css (hunk 1 above)
Comment (rohanmathur91) [reply documentation/style.css]: For a breakpoint almost at 800px, the main content was pushed behind the sidebar and was not properly seen, so I thought to make 800px as a breakpoint. Should I change it?

File: documentation/style.css (hunk 2 above)
Comment (rohanmathur91) [reply documentation/style.css]: Changed the high  z-index value.

File: index.html (hunk 3 above)
Comment (rohanmathur91) [reply index.html:18]: z-index, I followed the tailwind naming convention for that.

File: documentation/style.css (hunk 1 above)
Comment (smilegupta) [reply documentation/style.css]: if its the case of ipad you are handling 768px might also work @rohanmathur91

File: index.html (hunk 3 above)
Comment (smilegupta) [reply index.html:18]: awesome, wasnt aware of that. I learnt something new here :)

File: documentation/style.css (hunk 1 above)
Comment (rohanmathur91) [reply documentation/style.css]: Updated.

**Review by smilegupta — APPROVED** [review]
LGTM!

**Review by Aditi0505 — APPROVED** [review]
looks good to me.

**Pull request merged**